examples/dt/dcn/service-values.yaml

```diff
@@ -0,0 +1,261 @@
+        backend_availability_zone = az2
+  glance:
+    customServiceConfig: |
+      [DEFAULT]
+      enabled_backends = default_backend:rbd
+      [glance_store]
+      default_backend = default_backend
+      [default_backend]
+      rbd_store_ceph_conf = /etc/ceph/az0.conf
+      store_description = "RBD backend"
+      rbd_store_pool = images
+      rbd_store_user = openstack
+      rbd_thin_provisioning = True
+    glanceAPIs:
+      default:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = az0:rbd,az1:rbd,az2:rbd
+          [glance_store]
+          default_backend = az0
+          [az0]
+          rbd_store_ceph_conf = /etc/ceph/az0.conf
+          store_description = "az0 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [image_import_opts]
+          image_import_plugins = ['image_conversion']
+          [image_conversion]
+          output_format = raw
+        networkAttachments:
+          - storage
+        override:
+          service:
+            internal:
+              metadata:
+                annotations:
+                  metallb.universe.tf/address-pool: internalapi
+                  metallb.universe.tf/allow-shared-ip: internalapi
+                  metallb.universe.tf/loadBalancerIPs: 172.17.0.80
+              spec:
+                type: LoadBalancer
+        replicas: 3
+        type: split
+      az1:
+        customServiceConfig: |
+          [DEFAULT]
+          enabled_import_methods = [web-download,copy-image,glance-direct]
+          enabled_backends = az0:rbd,az1:rbd,az2:rbd
+          [glance_store]
+          default_backend = az1
+          [az1]
+          rbd_store_ceph_conf = /etc/ceph/az1.conf
+          store_description = "az1 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az0]
+          rbd_store_ceph_conf = /etc/ceph/az0.conf
+          store_description = "az0 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [az2]
+          rbd_store_ceph_conf = /etc/ceph/az2.conf
+          store_description = "az2 RBD backend"
+          rbd_store_pool = images
+          rbd_store_user = openstack
+          rbd_thin_provisioning = True
+          [image_import_opts]
```

Comment:
If Ceph is set as a backend, no need to explicitly enable `image_conversion` as the operator does that for you.
Remove L149-L152

Reply:
I'd need to test it again but I do not think It did, that's why we enabled it.

Reply:
I understood what you meant, I just believed It did not work - glance-operator did not set it so I tested it and you are right, I believed It did not work most likley because when an image was created no --import cmdline option was specified so the import method was not used for the image creation and that's why the conversion was not triggered. @sbekkerm sorry about that, we cna remove the image conversion lines.

Reply:
I've confirmed that there are no `image_import_opts` set